go.mod

```diff
@@ -89,6 +89,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
 
+require github.com/cespare/xxhash/v2 v2.1.2
```

Comment:
nit: Can we club this require statement with the above block?

Reply:
Unsure why that happened, now moved to the first require block